docs/extensions/readme.md

```diff
@@ -97,6 +98,7 @@ accountType:
   x-ms-enum:
     name: AccountType
     modelAsString: false
+    default: Standard_LRS
```

Comment:
I don't think we should have default on the `enum` definition itself, I think it belongs to the property.

We should have a call today and chat about it.